### ISSUE TITLE
Ansible recursive loop in template string during update

### ISSUE THREAD

**Issue by marijnschraagen** [proycon/LaMachine#120]
When running `lamachine-update`, Ansible fails with a recursive loop while registering metadata for `clam`. This is the same error as https://github.com/ansible/ansible/issues/39369.

I have tried running `lamachine-update force=1` which did not make a difference.

The installation was originally performed end of July 2018 without errors (with the attempted update using https://github.com/proycon/LaMachine/commit/6de4fae7a1dbc522df1ecc221dd98150a3b39e96).

Snippet of log (full log attached):

```
TASK [lamachine-register : Registering metadata: overview] ***********************************************************************
ok: [Trans-10] => (item=update) => {
    "update": "clam",
    "var_item": "update"
}
fatal: [Trans-10]: FAILED! => {}

MSG:

An unhandled exception occurred while templating '{{package}}'.
Error was a <class 'ansible.errors.AnsibleError'>, original message:
An unhandled exception occurred while templating '{{package}}'.
Error was a <class 'ansible.errors.AnsibleError'>,
...
Error was a <class 'ansible.errors.AnsibleError'>, original message:
An unhandled exception occurred while templating '{{package}}'.
Error was a <class 'ansible.errors.AnsibleError'>, original message:
recursive loop detected in template string: {{package}}
```
[lamachine-lmenv-20181005_110358.log](https://github.com/proycon/LaMachine/files/2449780/lamachine-lmenv-20181005_110358.log)

**Comment by proycon** [comment]
Confirmed, I'm getting this on all my test builds as well now suddenly. Will fix it straight away!

**Comment by proycon** [comment]
Should be solved now in v2.4.2, can you try a ``lamachine-update`` again?

(please reopen if it still fails)

**Comment by marijnschraagen** [comment]
This is indeed solved now, thanks! Although #117 still occurs, so the installation does not fully succeed yet.
